aiter/ops/triton/mha.py

```diff
@@ -2902,6 +2908,8 @@ def flash_attn_func(
         return_lse,
         return_attn_probs,
         torch.is_grad_enabled(),
+        fused_backward,
+        onekernel_backward,
```

Comment:
Remove these args from the top level API. Instead add a setter function which sets a global variable to control which kernel to use. See the MoE code for example https://github.com/ROCm/aiter/blob/main/aiter/ops/triton/moe_op.py#L18